packages/mui-internal-usage-tracker/README.md

```diff
@@ -0,0 +1,5 @@
+# @mui/internal-usage-tracker
```

Comment:
maybe:

```suggestion
# @mui/internal-download-tracker
```

so it feels less frightening from a privacy perspective. No, it doesn't include telemetry.

Reply:
I like the download tracker since it sounds less scary at a glance

Reply:
me too.

Reply:
`@mui/core-internal-download-tracker`, then?